doc/nrf/protocols/bt_mesh/configuring.rst

```diff
@@ -35,11 +35,113 @@ Mesh models
+* :kconfig:option:`CONFIG_HWINFO` - See :ref:`bt_mesh_dk_prov`, chapter UUID.
+* :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` - Use the Partition Manager for single image builds. See :ref:`partition_manager`
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_SETTINGS_STORAGE` - Size of the partition used for settings storage. This should most likely be increased.
 * :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` - Enables partial erase on supported hardware platforms.
   Partial erase allows the flash page erase operation to be split into several small chunks preventing longer CPU stalls.
   This improves responsiveness of a mesh node during the defragmentation of storage areas used by the settings subsystem.
+* :kconfig:option:`CONFIG_DK_LIBRARY` - Button and LED Library for the nRF5x development kits. Should be added if nRF5x development kits are used.
+
+Logging
+-------
+
+* :kconfig:option:`CONFIG_NCS_SAMPLE_DEFAULTS` - See :ref:`ug_logging_zephyr`
+* :kconfig:option:`CONFIG_LOG_MODE_DEFERRED` - Recommended to avoid slowing mesh message processing.
+  Helps improve LPN power consumption when friendship is established.
+* :kconfig:option:`CONFIG_LOG_BUFFER_SIZE` - Increase to avoid missing logs when complex protocol or functionality issue.
+* :kconfig:option:`CONFIG_LOG_PROCESS_THREAD_SLEEP_MS` - Decrease to flush logs more quickly.
+
+GATT Proxy performance
+----------------------
+
+* These options allow fitting full relayed mesh advertiser frame into single Link Layer payload.
+
+  * :kconfig:option:`CONFIG_BT_CTLR_DATA_LENGTH_MAX` =37
+  * :kconfig:option:`CONFIG_BT_BUF_ACL_TX_SIZE` =37
+
+* This option allows sending up to several data frames during the single connection interval.
+
+  * :kconfig:option:`CONFIG_BT_CTLR_SDC_TX_PACKET_COUNT` =10
+
+Bluetooth settings/performance
+------------------------------
+
+* :kconfig:option:`CONFIG_BT_COMPANY_ID` - Set the Bluetooth Company Identifier for this device.
+* :kconfig:option:`CONFIG_BT_DEVICE_NAME` - Bluetooth device name.
+* :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU` - Maximum L2CAP MTU for L2CAP TX buffers.
+* :kconfig:option:`CONFIG_BT_L2CAP_TX_BUF_COUNT` - Number of buffers available for outgoing L2CAP packets.
+* :kconfig:option:`CONFIG_BT_RX_STACK_SIZE` - Size of the receiving thread stack.
+* :kconfig:option:`CONFIG_BT_EXT_ADV_MAX_ADV_SET` - Maximum number of simultaneous advertising sets.
+* :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE` - Data size needed for HCI ACL RX buffers
+
+Disable unused Bluetooth features
+---------------------------------
+
+These features are disabled in the samples, but need to be considered if any of them is required by the application.
+
+* :kconfig:option:`CONFIG_BT_CTLR_DUP_FILTER_LEN` =0
+* :kconfig:option:`CONFIG_BT_CTLR_LE_ENC` =n
+* :kconfig:option:`CONFIG_BT_CTLR_CHAN_SEL_2` =n
+* :kconfig:option:`CONFIG_BT_CTLR_MIN_USED_CHAN` =n
+* :kconfig:option:`CONFIG_BT_CTLR_PRIVACY` =n
+* :kconfig:option:`CONFIG_BT_DATA_LEN_UPDATE` =n
```

Comment:
This should be `y`. Otherwise GATT Proxy performance improvements will not work.

Reply:
Why is this set to no in all our samples then? Should we remove this then from our samples?

Reply:
It has been already removed, hasn't it? Did I miss anything?

Reply:
Removed